### ISSUE TITLE
duplicate alerts not working properly

### ISSUE THREAD

**Issue by hprins** [betorvs/sensu-opsgenie-handler#10]
I am having issues with opsgenie subsequent alerts; meaning that my first alert gets to opsgenie the way it is supposed to, but if the alert remains open and a reminder alert is then triggered, the alert count remains at 1 (see attachment; x1, highlighted in yellow). The reminder goes into the Notes section (highlighted in orange) instead; this is not normal. I did check with Opsgenie support and they believe that the issue is not on their end.  Thanks.
![sensu_opsgenie_handler](https://user-images.githubusercontent.com/26799700/76328246-75a24380-62c1-11ea-8ca8-b551a08bec71.jpg)

**Comment by cwjohnston** [comment]
To drill into this further, both [sensu-plugins-opsgenie handler](https://github.com/sensu-plugins/sensu-plugins-opsgenie/blob/5.0.0/bin/handler-opsgenie.rb#L55-L57), and the closed-source [Sensu Enterprise integration](https://docs.sensu.io/sensu-enterprise/latest/integrations/opsgenie/) will  "create" an incident in OpsGenie every non-OK check result they receive. 

This has the effect of incrementing the ["deduplication" counter in OpsGenie's alert view](https://docs.opsgenie.com/docs/alerts-page), a behavior which some teams have come to rely on as an indicator of how many times this incident has reoccurred.

This handler [creates an incident in OpsGenie on the first non-OK check result](https://github.com/betorvs/sensu-opsgenie-handler/blob/0.0.10/main.go#L243-L245) for a given `entity/check` pair (aka event key). 

For each subsequent non-OK result for a given event key, this handler [adds a note to the existing incident](https://github.com/betorvs/sensu-opsgenie-handler/blob/0.0.10/main.go#L250-L252). Adding a note does not have the effect of incrementing the deduplication counter in OpsGenie's alerts view, leading to some confusion as to whether a given event key has occurred more than once.

Edit: Finally, unlike the Sensu Enterprise integration, the sensu-plugins-opsgenie handler does  not record event data as notes on an incident.

**Comment by cwjohnston** [comment]
@betorvs would you be open to a PR which changes the default behavior of the handler? Or would you prefer a change to be gated by a command-line flag?

**Comment by betorvs** [comment]
@cwjohnston i liked to give to user this option. But as @hprins told us this is not expected by opsgenie. In these case, we can switch to this option to send a alert instead of adding a new note. 
Do you have this changes @cwjohnston already done? I will appreciate.

**Comment by betorvs** [comment]
Hi @hprins please test new version 0.1.0. I changed the sensu-opsgenie-handler to send recurrence events as alerts and not as note. 
I tested with my event*and json here and works perfectly. 
If everything is ok, I will close this issue.

**Comment by hprins** [comment]
Hi @betorvs ; it works fine now. Thanks
